lib/src/presentations/forgot_password/screen/forgot_password_view.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:jobspot/src/core/common/custom_toast.dart';
+import 'package:jobspot/src/core/config/localization/app_local.dart';
+import 'package:jobspot/src/core/constants/constants.dart';
+import 'package:jobspot/src/core/extension/string_extension.dart';
+import 'package:jobspot/src/core/function/loading_animation.dart';
+import 'package:jobspot/src/core/resources/data_state.dart';
+import 'package:jobspot/src/presentations/forgot_password/cubit/forgot_password_cubit.dart';
+import 'package:jobspot/src/presentations/forgot_password/domain/forgot_password_coordinator.dart';
+import 'package:jobspot/src/presentations/sign_in/widgets/custom_button.dart';
+import 'package:jobspot/src/presentations/sign_in/widgets/custom_title_text_input.dart';
+
+class ForgotPasswordView extends StatelessWidget {
+  const ForgotPasswordView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: BlocListener<ForgotPasswordCubit, ForgotPasswordState>(
+        listenWhen: (previous, current) {
+          if (previous.isLoading) Navigator.of(context).pop();
+          return true;
+        },
+        listener: (context, state) {
+          if (state.isLoading) loadingAnimation(context);
+
+          if (state.dataState is DataFailed) {
+            customToast(context, text: state.dataState?.error ?? "");
+          }
+
+          if (state.dataState is DataSuccess) {
```

Comment:
the same above

Reply:
In this case `state.dataState` can be `null` so it needs to be checked for `null` before calling it. The command will be longer!